assets/handy-window-dark.css

```diff
@@ -0,0 +1,7 @@
+window.csd.unified:not(.solid-csd):not(.fullscreen):not(.maximized) decoration-overlay {
+  box-shadow: inset 0 0 0 1px rgba(255, 255, 255, 0.07);
+}
+
+window.csd.unified:not(.solid-csd):not(.fullscreen):not(.maximized) decoration-overlay~* headerbar {
```

Comment:
I just thought that the selector doesn't have to be that strange to work, as we are sure this only apply to libhandy apps:

```diff
- window.csd.unified:not(.solid-csd):not(.fullscreen):not(.maximized) decoration-overlay~* headerbar {
+ window.csd.unified:not(.solid-csd):not(.fullscreen):not(.maximized) headerbar {
```

We can also remove the default dark outline which is no more needed thanks to the bright one:
```css
decoration {
  box-shadow: 0 3px 9px 1px rgba(0,0,0,.5);
}
```
This will overwrite these lines: https://github.com/ubuntu/yaru/blob/21f6d2b0546fa49bdc1969c93ffd92abb3f4141b/gtk/src/default/gtk-3.0/_common.scss#L4556-L4557

Reply:
like this?
```css
window.csd.unified:not(.solid-csd):not(.fullscreen):not(.maximized) decoration-overlay {
  box-shadow: inset 0 0 0 1px rgba(255, 255, 255, 0.07);
}

window.csd.unified:not(.solid-csd):not(.fullscreen):not(.maximized) headerbar {
  box-shadow: none;
}

decoration {
  box-shadow: 0 3px 9px 1px rgba(0, 0, 0, .5);
}

decoration:backdrop {
  box-shadow: 0 3px 9px 1px transparent,
    0 2px 6px 2px rgba(0, 0, 0, .2);
}
```

Reply:
It seems good for me :+1: